test/unit/builtins/string.spec.ts

```diff
@@ -99,6 +103,8 @@ test.each([
     { inp: "hello test", searchValue: "t", offset: 6 },
     { inp: "hello test", searchValue: "t", offset: 7 },
     { inp: "hello test", searchValue: "h", offset: 4 },
+    { inp: "hello test", searchValue: "o", offset: -1 },
```

Comment:
Maybe it's nicer to use a searchValue that actually appears in the string multiple times (to show it returns the right one)?